app/views/profile/show.html.erb

```diff
@@ -2,35 +2,37 @@
   <%= render 'users/header' %>
 
   <!--main content-->
-  <% unless @user.profile.title.blank? %>
-    <h3 class="heading"><%=h @user.profile.title %></h3>
+  <% if @profile.title.present? %>
+    <h3 class="heading"><%=h @profile.title %></h3>
   <% end %>
 
   <div class="wrapper">
     <dl class="meta">
-      <dt class="pseuds"><%= link_to ts("My pseuds:"), user_pseuds_path(@user) %></dt>
-      <dd class="pseuds"><%= print_pseud_list(@user.pseuds) %></dd>
+      <dt class="pseuds">
+        <%= link_to ts("All my pseuds (%{pseud_number})", :pseud_number => @user.pseuds.count), user_pseuds_path(@user) %>
```

Comment:
I think the phrasing here is confusing. By having `All my pseuds (#)` as the `dt` and a list of pseuds as the `dd`, it seems to be implying that all of the pseuds are included in the list.

That's why I suggested adding `and # more pseuds` (or some similar text) at the end of the pseud list. It's hard to quickly count how many pseuds are displayed. If someone has 15 or 25 pseuds, it's going to be difficult to know immediately whether the list includes all of them, or whether the viewer need to click the link to see the rest.

Reply:
Yeah, I meant put the "All Pseuds (#)" text at the end rather than changing the text in the `<dt>`, but you make a good point about the math. 

However, I'd expect something like "and X more" to expand the list to show the remaining pseuds (like the "x more users" link in kudos or the "See all" link in the "Signed up" section of [challenge profiles](https://archiveofourown.org/collections/yuletide2015/profile)), not take me to a separate page. 

Maybe something more like "(All pseuds, including X more)"?

Reply:
I would prefer to have a clear decision handed to me, my decision-making cognitive tank is depleted.

Reply:
You can just use my previous suggestion since I am also similarly exhausted.